Sources/LiveViewNative/Views/Text Input and Output/TextEditor.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  TextEditor.swift
+//  LiveViewNative
+//
+//  Created by Shadowfacts on 2/8/23.
+//
+
+import SwiftUI
+
+struct TextEditor<R: CustomRegistry>: TextFieldProtocol {
+    @ObservedElement var element: ElementNode
+    let context: LiveContext<R>
+    @FormState var value: String?
+    @FocusState private var isFocused: Bool
+    @LiveBinding(attribute: "find-presented") private var isFindPresented = false
+    
+    init(element: ElementNode, context: LiveContext<R>) {
+        self.context = context
+    }
+    
+    var body: some View {
+        SwiftUI.TextEditor(text: textBinding)
```

Comment:
FYI, TextEditor is completely unavailable on watchOS

Reply:
What about for ants?

Reply:
🤦‍♂️ I completely missed that